config/initializers/app_info.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 module AppInfo
-  GIT_REVISION = `git rev-parse HEAD`&.chomp
+  GIT_REVISION = ENV['GIT_REVISION']
```

Comment:
I think you might want to try it with a default `ENV.fetch('GIT_REVISION', 'MISSING_GIT_REVISION') or similar.

Reply:
I think as per discussion in Slack this is fine to expect a value here - if it's `nil`/blank that means the BRD cycle is problematic and likely this would be the least of our concerns

Reply:
Thought about this some more -- its going to be blank in development and test too. Let's put some kind of default in.

Reply:
It won't be blank, lower envs are part of the build/release process, unless I'm misunderstanding what we're talking about when we say build/release.

The image ref tag *has* to be present, I think, else Ansible doesn't know what to build.

Anyways I added a default value feel free to change it if there's something more useful.

Reply:
At least until we don't have ansible 😄 

Updated.

Reply:
NBD either way IMO, they a default val or `nil` will both work just fine.